Geometry/MTDCommonData/src/BTLNumberingScheme.cc

```diff
@@ -0,0 +1,130 @@
+#include "Geometry/MTDCommonData/interface/BTLNumberingScheme.h"
+#include "DataFormats/ForwardDetId/interface/BTLDetId.h"
+
+#include <iostream>
+#include <cstring>
+
+//#define DEB_NUM
```

Comment:
Considering doing this in a more standard way (i.e. `EDM_ML_DEBUG`, `LogDebug`)

Reply:
DEB_NUM turned into EDM_ML_DEBUG